lib/ead/traject/ead2_config.rb

```diff
@@ -77,7 +77,7 @@ def eadid_from_url_or_text(field_name)
     ead_id.sub!(/^ldpd_/,'')
     ead_id.gsub!(/[^A-Za-z0-9]/,'-')
     if ead_id
-      if ead_id =~ /^\d/
+      if ead_id =~ /^\d/ || ead_id =~ /^in\d/
```

Comment:
```suggestion
      if ead_id =~ /^(in)*\d/
```
What you have works, but you could also do this so that it's just one regular expression check.